src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/TestUtil.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import java.text.SimpleDateFormat
+
+class TestUtil private constructor() {
+  companion object {
+    fun objectMapper(): ObjectMapper {
+      val mapper = ObjectMapper()
+      mapper.registerModule(JavaTimeModule())
+      mapper.dateFormat = SimpleDateFormat("yyyy-MM-dd")
```

Comment:
I have tended to use ISO Date format rather than simpleDateFormat

```
formatter = DateTimeFormatter.ISO_LOCAL_DATE
```

Reply:
This is a different date format class.

`java.text.DateFormat`

not

`java.time.format.DateTimeFormatter`

Reply:
Fair enough!